### ISSUE TITLE
Support ESLint 8.x

### ISSUE THREAD

**Issue by MichaelDeBoey** [mysticatea/eslint-plugin-node#294]
ESLint has [released](https://eslint.org/blog/2021/08/eslint-v8.0.0-beta.0-released) the first beta version of v8 🎉

It would be awesome to have official ESLint 8 support. 👊
I'm happy to help where I can of course 🙂

---

Follow-up of #223

**Comment by aladdin-add** [comment]
for some blocked by the plugin, I've just pushed a fork: [eslint-plugin-n](https://www.npmjs.com/package/eslint-plugin-n).

it just added supports for eslint v8.

**Comment by voxpelli** [comment]
@aladdin-add Can you summarize somewhere on how it relates to the ESLint 8 PR:s in this repo? What the changes are from upstream? + maybe links to PR:s for those changes?

Great to have a fork to use until upstream gets updated 👍

**Comment by MichaelDeBoey** [comment]
@aladdin-add Would be nice if we could transfer repos  like this one to a new @eslint-community org

CC/ @mysticatea

**Comment by sosukesuzuki** [comment]
Thank you. I agree with @MichaelDeBoey. It would be nice to be able to take over maintenance and release from @mysticatea

**Comment by voxpelli** [comment]
As we need them for StandardJS as well, I think we can maybe add them there if we want a common org as well

**Comment by MichaelDeBoey** [comment]
Transferring them to the @standard org is fine for me too 🤷‍♂️

Just need to get in touch with @mysticatea somehow to get it done and then I think a lot of PRs can get merged quickly

**Comment by aladdin-add** [comment]
nicholas also agreed to the eslint-community, will put together an RFC for how it will work.

**Comment by MichaelDeBoey** [comment]
Let me know if I can help somehow!

**Comment by tennox** [comment]
_shy bump :sweat:_

**Comment by niksy** [comment]
It seems like [eslint-plugin-n](https://github.com/weiran-zsd/eslint-plugin-node) is maintained version of this plugin.

**Comment by voxpelli** [comment]
[`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n) is the maintained version of this module.

We switched to it in eg. [`eslint-config-standard`](https://github.com/standard/eslint-config-standard) / [`standard`](https://github.com/standard/standard) and it is maintained by me and other members of the official [ESLint community](https://github.com/eslint-community) organization.
